index.html

```diff
@@ -240,5 +240,74 @@ <h2>Let's reserve <span>a table</span></h2>
                 </div>
             </div>
         </section>
+        <section id="chefs-section">
+            <div class="chefs-section__container">
+                <h2>Our Greatest <span>Chefs</span></h2>
+                <div class="chefs-section__chefs">
+                    <div class="chefs-section__chef">
+                        <img src="./assets/betran.png" alt="Betran chef" />
+                        <h4>Betran Komar</h4>
+                        <p>Head chef</p>
+                    </div>
+                    <div class="chefs-section__chef">
+                        <img src="./assets/ferry.png" alt="Ferry chef" />
+                        <h4>Ferry Sauwi</h4>
+                        <p>Chef</p>
+                    </div>
+                    <div class="chefs-section__chef">
+                        <img src="./assets/iswan.png" alt="Iswan chef" />
+                        <h4>Iswan</h4>
+                        <p>Chef</p>
+                    </div>
+                </div>
+                <button class="btn-large-orange">View All</button>
+            </div>
+        </section>
+        <section id="testimonials-section">
+            <div class="testimonials-section__container">
+                <h2>What customers say</h2>
+                <div class="testimonials-section__testimonials">
+                    <div class="testimonials-section__testimonial">
+                        <img src="./assets/customer-1.png" alt="" />
+                        <h4>James Steven</h4>
```

Comment:
Typo: The name of the person is not James Steven. It's actually James Maden.

Reply:
ok got it

Reply:
lgtm